pkg/crd/testdata/cronjob_types.go

```diff
@@ -381,6 +381,11 @@ type CronJobSpec struct {
 
 	// This tests that embedded struct, which is an alias type, is handled correctly.
 	InlineAlias `json:",inline"`
+
+	// Test that we can add a field that can only be set on updates using XValidation OptionalOldSelf.
+	// The validation is applied to the spec struct itself and not the field.
```

Comment:
I'm a bit confused about this statement. Is it correct?

Reply:
Whoops - forgot to update that after I decided to move it from the spec. Will update that soon

Reply:
Fixed in the latest push.